examples/chakra-ui/package.json

```diff
@@ -2,20 +2,20 @@
 	"name": "@conform-example/chakra-ui",
 	"private": true,
 	"dependencies": {
-		"@chakra-ui/react": "^2.4.2",
-		"@conform-to/react": "1.1.0-pre.0",
-		"@emotion/react": "^11.10.5",
-		"@emotion/styled": "^11.10.5",
-		"framer-motion": "^7.6.19",
-		"react": "^18.2.0",
-		"react-dom": "^18.2.0",
+		"@chakra-ui/react": "2.8.2",
+		"@conform-to/react": "workspace:*",
+		"@emotion/react": "11.11.4",
+		"@emotion/styled": "11.11.5",
+		"framer-motion": "11.0.24",
+		"react": "18.2.0",
+		"react-dom": "18.2.0",
 		"react-scripts": "5.0.1"
```

Comment:
I've pinned all deps to a specific version, and when we want to upgrade we can run `pnpm update -r --interactive --latest` and it'll swap out all the deps.

pinning versions is imo the best way to build software because you don't have random version bumps come into the project to screw things up